app/views/spree/shared/_rating.html.erb

```diff
@@ -1,15 +1,72 @@
-<% stars = product.stars %>
-<% reviews_count = product.reviews_count %>
+<% @average_rating = product.reviews.count > 0 ? product.reviews.sum(:rating).to_f / product.reviews.count : 0  %>
+
+<% reviews_count  = product.reviews.count %>
+<% raiting_five   = reviews_count != 0  ? product.reviews.raiting_five * 100 / reviews_count : 0 %>
```

Comment:
make a helper for this

raiting_four   = reviews_count != 0  ? product.reviews.raiting_four * 100 / reviews_count